test/unit/StreamrClient.test.js

```diff
@@ -587,6 +603,14 @@ describe('StreamrClient', () => {
                 })
             })
 
+            it('should add a new SubscribedStream', () => {
+                const streamId = 'streamId'
+                connection.expect(new SubscribeRequest(streamId))
+                client.subscribe(streamId, () => {})
+                const stream = client.subscribedStreams.streamId
```

Comment:
Digging into module internals isn't a great pattern. Is there publicly exposed functionality we can test instead? Is this already covered by existing tests?